### PR TITLE
chore: add gRPC for querying incentivized packets for a specific channel

### PR DESCRIPTION
**Pull request by colin-axner**
## Description

ref: #970

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: modules/apps/29-fee/keeper/grpc_query.go
Comment (colin-axner) [inline modules/apps/29-fee/keeper/grpc_query.go:72]: no test case because the tests will panic on a nil request

File: modules/apps/29-fee/keeper/grpc_query.go
Comment (colin-axner) [inline modules/apps/29-fee/keeper/grpc_query.go:83]: left off a test case because then I'd have to set an invalid key. I could do this, but the correctness of keeper setting shouldn't really be tested by grpc query tests

File: proto/ibc/applications/fee/v1/query.proto
Comment (damiannolan) [inline proto/ibc/applications/fee/v1/query.proto:67]: I'd like to clean up the protodoc comments to document each field similarly to this one. Just a concise lowercase comment for each field in the proto message. 

I can open an issue and we can address it after this is merged!

Comment (damiannolan) [inline proto/ibc/applications/fee/v1/query.proto:67]: I'd like to clean up the protodoc comments to document each field similarly to this one. Just a concise lowercase comment for each field in the proto message. 

I can open an issue and we can address it after this is merged!

File: proto/ibc/applications/fee/v1/query.proto
Comment (damiannolan) [inline proto/ibc/applications/fee/v1/query.proto:77]: Should this be non-nullable?

**Review by damiannolan — APPROVED** [review]
Nice, we should be looking pretty good for an alpha release next week.
I think the queries are the main thing still on the plate, and maybe some test cleanup :)

**Comment by damiannolan** [comment]
Apologies for the conflicts!

File: proto/ibc/applications/fee/v1/query.proto
Comment (colin-axner) [reply proto/ibc/applications/fee/v1/query.proto:77]: I thought so? But didn't we have conversation on the gRPC's needing to be pointers?

Comment (damiannolan) [reply proto/ibc/applications/fee/v1/query.proto:77]: Hmm, I do vaguely recall something. I can't remember why, or think of a reason it needs to a pointer type right now tho! Happy to leave as is, your call!

**Review by seantking — APPROVED** [review]

**Pull request merged**
